shared/teams/team/member/add-to-channels.tsx

```diff
@@ -322,17 +322,6 @@ const SelfChannelActions = (p: {
   const [waiting, setWaiting] = React.useState(false)
   const stopWaiting = React.useCallback(() => setWaiting(false), [])
 
-  const updatedTrigger = useEditState(s => s.editUpdatedTrigger)
```

Comment:
this causes a lot of thrash as each row will try and reload everything and the flow is wonky. Instead we expose the async of the save calls and then close if it works